File: Selenium/Activities/activity19.java
import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.Alert;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.support.ui.WebDriverWait;

import java.time.Duration;

public class activity19 {
    public static void main(String[] args) {
        System.setProperty(FirefoxDriver.SystemProperty.BROWSER_LOGFILE, "/dev/null");
        WebDriverManager.firefoxdriver().setup();
        WebDriver driver = new FirefoxDriver();
        WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(10));
        driver.get("https://www.training-support.net/selenium/javascript-alerts");
        System.out.println("The title of the page is: " +driver.getTitle());
        //Find the button to open a CONFIRM alert and click it.
        driver.findElement(By.id("confirm")).click();
        //Switch the focus from the main window to the Alert box and get the text in it and print it.
        Alert confirmAlert = driver.switchTo().alert();
        String alertText = confirmAlert.getText();
        System.out.println("The message in the confirmation alert box is: " + alertText);
        //Close the alert once with Ok and again with Cancel.
        confirmAlert.accept();
        System.out.println("The alert is accepted and closed");
        driver.findElement(By.id("confirm")).click();
        Alert confirmAlert1 = driver.switchTo().alert();
        String alertText1 = confirmAlert1.getText();
        System.out.println("The message in the confirmation alert box is: " + alertText1);
        confirmAlert1.dismiss();
        System.out.println("The alert is cancelled and closed");
        driver.quit();
    }
}

File: Selenium/Activities/Activity19.py
from selenium import webdriver
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager
from selenium.webdriver.remote.webdriver import By
from selenium.webdriver.support.select import Select

service = FirefoxService(GeckoDriverManager().install())

with webdriver.Firefox(service=service) as driver:
    driver.get("https://www.training-support.net/selenium/javascript-alerts")
    print("Page title is: ", driver.title)
    #Find the button to open a CONFIRM alert and click it.
    driver.find_element(By.ID, "confirm").click()
    #Switch the focus from the main window to the Alert box and get the text in it and print it.
    confirm_alert = driver.switch_to.alert
    print(confirm_alert.text)
    #Close the alert once with Ok and again with Cancel.
    confirm_alert.accept()
    print("The alert is accepted")
    driver.find_element(By.ID, "confirm").click()
    confirm_alert1 = driver.switch_to.alert
    print(confirm_alert1.text)
    confirm_alert1.dismiss()
    print("The alert is dismissed")